test/required.test.ts

```diff
@@ -464,7 +464,7 @@ describe('Required helper', () => {
         assert.strictEqual(result, true, 'returned result should be true');
         assert.strictEqual(tryExecuteCommandMock.callCount, 3, 'tryExecuteCommand should be called 3 times');
         assert.strictEqual(showErrorMessageMock.called, false, "showErrorMessage shouldn't be called");
-        assert.strictEqual(executeVSCommandMock.called, false, "executeVSCommand shouldn't be called");
+        assert.strictEqual(executeVSCommandMock.called, true, "executeVSCommand shouldn't be called");
```

Comment:
this test is either wrong or the assertion statement is incorrect. Can you fix it.

Reply:
Hey @michaeljohnbennett, thanks for that :)

Now, we are executing commands inside the Output class `(commands.executeCommand)` so i forgot to change the assertion text to: `"executeVSCommand should be called"`

Now, it's fixed. Thank you again!